x/gamm/pool-models/balancer/codec.go

```diff
@@ -26,6 +26,7 @@ func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
 	registry.RegisterImplementations(
 		(*sdk.Msg)(nil),
 		&MsgCreateBalancerPool{},
+		&PoolParams{},
```

Comment:
PoolParams doesnt implement the message interface though?

Reply:
yes, I have added PoolParams directly to codec like your suggestion

Reply:
I think what @ValarDragon was saying was that `PoolParams` does not follow implementations of proto.Message. 

Structs should be implemented upon specific interfaces via `RegisterImplementations`, @ValarDragon  do you think we should make an interface for pool params?